src/list.rs

```diff
@@ -23,7 +23,7 @@ pub struct FileInArchive {
 
 /// Actually print the files
 pub fn list_files(archive: &Path, files: Vec<FileInArchive>, list_options: ListOptions) {
-    println!("{}:", archive.display());
+    println!("Archiv: {}", archive.display());
```

Comment:
Is Archiv a typo?

Reply:
yeah, that should have been "Archive: " :D